core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -4223,10 +4234,12 @@ public void visit(RelNode node, int ordinal, @Nullable RelNode parent) {
     private final boolean trim;
     private final UnaryOperator<SqlToRelConverter.Config> config;
     private final SqlConformance conformance;
+    private final boolean query;
+
```

Comment:
Why we add a flag there ? Can we eliminate that.

Reply:
I  want to use it to mark the input string , like the inner class Sql of SqlValidatorTest used.

Reply:
and we can use like **expr().ok** to test expression later. so I think maybe we need the flag. and thanks for your reply  danny

Reply:
@danny0405  Please take a look. Thanks.